README.md

```diff
@@ -53,6 +53,45 @@ $ docker run -it --rm --name my-running-script -v "$PWD":/usr/src/app -w
 /usr/src/app node:4 node your-daemon-or-script.js
 ```
 
+## Verbosity
+
+By default Node.js Docker Image has NPM log verbosity set to `verbose` instead
+of the default `warn`. This is because of the way Docker is isolated from the
+host operating system and you are not guaranteed to be able to retrieve the
+`npm-debug.log` file when something breaks.
```

Comment:
The layer won't be thrown away on a failed docker build; the container is still there in `docker ps -a`. It is just not easy to jump inside the stopped container. `docker cp`, `docker inspect`, and `docker logs` are all we can really do to debug it.

Maybe we'll get the ability to jump into a stopped container: https://github.com/docker/docker/pull/19746.